ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -192,7 +214,9 @@ open class NavigationBar: UINavigationBar {
     private let contentStackView = ContentStackView() //used to contain the various custom UI Elements
     private let rightBarButtonItemsStackView = UIStackView()
     private let leftBarButtonItemsStackView = UIStackView()
-    private let spacerView = UIView() //defines the space between the left and right barbuttonitems stack
+    private let leadingSpacerView = UIView()
```

Comment:
we should still have comments. don't think it is that obvious. Also, why have private let for leadingspacerview but create it locally for trailingspacerview?

Reply:
To be consistent, I'll add trailingspacerview here. I'll also add back the comment. The reason why I didn't include trailingspacerview was because it isn't referenced anywhere else than the method where it is added to the view hierarchy.